Xamarin.Forms.Platform.iOS/Renderers/ListViewRenderer.cs

```diff
@@ -666,13 +666,16 @@ internal nfloat GetEstimatedRowHeight(UITableView table)
 				}
 
 				// We're going to base our estimate off of the first cell
-				Cell firstCell;
 				var isGroupingEnabled = List.IsGroupingEnabled;
 
 				if (isGroupingEnabled)
-					firstCell = templatedItems.ActivateContent(0, templatedItems.GetGroup(0)?.ListProxy[0]);
-				else
-					firstCell = templatedItems.ActivateContent(0, templatedItems.ListProxy[0]);
+					templatedItems = templatedItems.GetGroup(0);
+
+				object _firstCell = null;
```

Comment:
as fucked up as it is, our Style recommend using `_xxx` for instance fields, not local variables.